### DISCUSSION TITLE
Feature request: bump search-insights version to 2.17.2

### DISCUSSION THREAD

**Discussion by ahello-pass** [algolia/instantsearch#6432] [Ideas]
Bump search-insights version here https://github.com/algolia/instantsearch/blob/76b0247108a0597809a543a8c925aa7ae14c6b18/packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts#L44 as 2.17.2 version fixes older browser compatibility problem on local storage https://github.com/algolia/search-insights.js/releases/tag/v2.17.2

**Comment by Haroenv** [comment]
Good point, we'll do that. Just FYI you can also manually choose the newer version by passing `insightsClient` and loading it yourself.

**Reply by ahello-pass** [reply]
Thank you !!
